packages/utils/src/internals/error_tracker.ts

```diff
@@ -366,6 +379,13 @@ export class ErrorTracker {
         return result.sort((a, b) => b[0] - a[0]).slice(0, count);
     }
 
+    async captureSnapshot(storage: Record<string, unknown>, error: ErrnoException, context: CrawlingContext) {
+        const { screenshotFilename, htmlFilename } = await this.errorSnapshotter.captureSnapshot(error, context);
+
+        storage.firstErrorScreenshot = screenshotFilename;
+        storage.firstErrorHtmlSnapshot = htmlFilename;
```

Comment:
I would append `Url` to both so `firstErrorScreenshotUrl` and `firstErrorHtmlSnapshotUrl`